handlers/graph.go

```diff
@@ -379,7 +379,7 @@ func buildNamespaceTrafficMap(namespace string, o options.Options, client *prome
 	populateTrafficMapTcp(trafficMap, &tcpUnkVector, o)
 
 	// 2) query for traffic originating from a workload outside of the namespace. Exclude any "unknown" source telemetry (an unusual corner case)
-	tcpGroupBy = "source_workload_namespace,source_workload,source_app,source_version,destination_service_namespace,destination_service_name,destination_workload,destination_app,destination_version"
+	// tcpGroupBy = "source_workload_namespace,source_workload,source_app,source_version,destination_service_namespace,destination_service_name,destination_workload_namespace,destination_workload,destination_app,destination_version"
```

Comment:
why is this commented out?

Reply:
I'll remove it.  With the changes we use the same groupBy for the different TCP queries.